subprojects/docs/src/docs/userguide/authoring-builds/incremental_builds.adoc

```diff
@@ -37,9 +44,8 @@ What this means for build authors is simple: you need to tell Gradle which task
 
 Let’s now look at how you can register task properties as inputs and outputs.
 
-
 [[sec:task_input_output_annotations]]
-=== Custom task types
+== Input and output declaration
```

Comment:
🤔 There are two ways to declare the inputs and outputs of a task: Using annotations on custom task types or using the runtime API. This section is about using annotations. I think if we change this section name then we should also change the `Runtime API` section name.

Reply:
Clarified by adding "via annotations" to the title